e2e-runner/lib/darwin/runner.sh

```
@@ -132,7 +132,7 @@ userProfile="$HOME"
 toolsInstallDir="$userProfile/tools"
 
 # Output file for built podman desktop binary
-outputFile="pde2e-binary-path.log"
+outputFile="binary-path.log"
```

Shellcheck (0.11.0)

[warning] outputFile appears unused. Verify use (or export if used externally).

(SC2034)